internal/namespaces/instance/v1/custom_marshal.go

```diff
@@ -325,23 +325,25 @@ func (sg *customSecurityGroupResponse) MarshalHuman() (out string, err error) {
 	if err != nil {
 		return "", err
 	}
+	defaultInboundPolicy = terminal.Style(defaultInboundPolicy, color.Bold)
```

Comment:
If the whole title is bold, we should try to use `color.Bold` only once per title. 

Wouldn't something like that work ? 
```
inboundRulesTitle := terminal.Style("Inbound Rules (default policy " + defaultInboundPolicy + "):", color.Bold)
```

and at line 340 

```
inboundRulesView := inboundRulesTitle + "\n" + inboundRulesContent

Reply:
Nop, the `defaultInboundPolicy` (in your fist code section) will escape at the end and the `):` will not be bold. :/

Reply:
And what about ?
```
inboundRulesTitle := "Inbound Rules (default policy " + defaultInboundPolicy + "):"
inboundRulesView := terminal.Style(inboundRulesTitle, color.Bold) + "\n" + inboundRulesContent

Reply:
ho, ok, defaultInboundPolicy is already colored

Sorry, i missed that

Reply:
Can you add a comment please ? 

It's not clear `defaultInboundPolicy` is already colored, wich leads to the escaping problem.